Cannot build on Windows

**Describe the bug**
\src\mumble\GlobalShortcut_win.cpp(233): error C2065: 'InputGkey': undeclared identifier
\src\mumble\GlobalShortcut_win.cpp(233): error C2672: 'qMetaTypeId': no matching overloaded function found
src\mumble\GlobalShortcut_win.cpp(233): error C2974: 'qMetaTypeId': invalid template argument for 'T', type expected
vcpkg\installed\x64-windows-static-md\include\qt5\QtCore/qmetatype.h(1929): note: see declaration of 'qMetaTypeId'
NMAKE : fatal error U1077: '"C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\Common7\IDE\CommonExtensions\Microsoft\CMake\CMake\bin\cmake.exe"' : return code '0x2'


**Steps to Reproduce**
cmake --build .

**Desktop (please complete the following information):**
 - OS: win 10
 - Version: latest and also from archive 48 commits before

.

--- a/src/mumble/GlobalShortcut_win.cpp
+++ b/src/mumble/GlobalShortcut_win.cpp
@@ -229,8 +229,14 @@ QList< Shortcut > GlobalShortcutWin::migrateSettings(const QList< Shortcut > &ol
 			}
 
 			if (button.userType() == qMetaTypeId< InputHid >() || button.userType() == qMetaTypeId< InputKeyboard >()
-				|| button.userType() == qMetaTypeId< InputMouse >() || button.userType() == qMetaTypeId< InputXinput >()
-				|| button.userType() == qMetaTypeId< InputGkey >()) {
+				|| button.userType() == qMetaTypeId< InputMouse >()
+#ifdef USE_XBOXINPUT
+				|| button.userType() == qMetaTypeId< InputXinput >()
+#endif
+#ifdef USE_GKEY
+				|| button.userType() == qMetaTypeId< InputGkey >()
+#endif
+			) {
 				// Already in the new format.
 				continue;
 			}

